config/locales/en.yml

```diff
@@ -1137,7 +1137,10 @@ en:
     services:
       create:
         notice: "%{resource_type} created."
-        error: "Couldn't create %{resource_type}. Check your Plan limits" # TODO: this is not always true... there are other reasons of failure
+        errors:
+          name: "Name can't be blank."
```

Comment:
The problem of this approach is that we have to rewrite all possible errors of all validations in here, so when we add/edit/remove validations from service, we also have to change it here 😞

Reply:
It's not the only approach, that's for free. Either we leave it as it is, or roll back to the default ugly one or remove the entire flash message. There's also feedback per field.

Reply:
> Either we leave it as it is, or roll back to the default ugly one or remove the entire flash message

I don't think so :smile: I will write the alternative I have in mind this afternoon :wink:

Reply:
Bare in mind there's already implemented the per field error feedback, the flash message is complementary

Reply:
@didierofrivia https://github.com/3scale/porta/pull/1437 is pointing to this PR to make clearer what we mean 😄